wpilibj/src/generate/pwm_motor_controllers.json

```diff
@@ -168,5 +168,19 @@
       "deadbandMin": 1.480,
       "min": 0.997
     }
+  },
+  {
+    "name": "Koors40",
+    "Manufacturer": "AndyMark",
+    "DisplayName": "Koors40",
+    "ResourceName": "Koors40",
+    "PeriodMultiplier": 1,
```

Comment:
Multiplier has to be 4x, other than that timings look pretty reasonable on samples I have.

Reply:
```suggestion
    "PeriodMultiplier": 4,
```